code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -138,6 +138,13 @@
 		if(!length(candidates))
 			break
 		var/mob/applicant = pick(candidates)
+
+		var/depth_counter
+		while(isobserver(applicant)&&prob(50)&&depth_counter<10)
```

Comment:
Only observers can press yes to this dialogue box making this PR do nothing

Reply:
Adressed in the next commit